source/main/trezor/connection.ts

```diff
@@ -7,14 +7,25 @@ export const initTrezorConnect = async () => {
     await TrezorConnect.init({
       popup: false, // render your own UI
       webusb: false, // webusb is not supported in electron
-      debug: true,
+      debug: process.env.DEBUG_TREZOR === 'true',
       manifest,
     });
+
     logger.info('[TREZOR-CONNECT] Called TrezorConnect.init()');
   } catch (error) {
     logger.info('[TREZOR-CONNECT] Failed to call TrezorConnect.init()');
     throw error;
   }
 };
 
-export const reinitTrezorConnect = initTrezorConnect;
+export const reinitTrezorConnect = () => {
+  try {
+    logger.info('[TREZOR-CONNECT] Called TrezorConnect.dispose()');
+    TrezorConnect.dispose();
```

Comment:
From 8.2.8, `trezor-connect` no longer allows multiple `TrezorConnect.init()` calls - `.dispose()` must be called first. Unfortunately, sometimes it throws exceptions so I had to mute those.